src/index.ts

```diff
@@ -269,36 +269,49 @@ class BytenodeWebpackPlugin implements WebpackPluginInstance {
   }
 }
 
-function prepare(context: string | undefined, location: string | string[], name?: string, suffix = ''): Prepared {
+function prepare(context: string | undefined, location: string | string[] | { import: string } | { import: string }[], name?: string, suffix = ''): Prepared {
```

Comment:
It might make sense to have Webpack4Location and Webpack5Location types to help document these structures?